lib/openam.js

```diff
@@ -149,15 +156,18 @@ OpenAMClient.prototype.getCDSSOUrl = function (target, provider) {
  * to have the REST calls for policy evaluation privilege in OpenAM.
  *
  * @param {object} params Policy request params {@see https://backstage.forgerock.com/#!/docs/openam/current/dev-guide#rest-api-authz-policy-decisions}
+ * @param {string} [realm=/]
  * @param {string} sessionId OpenAM session ID
  * @param {string} cookieName OpenAM session cookie name
  * @return {Promise} Policy decision response
  */
-OpenAMClient.prototype.getPolicyDecision = function (params, sessionId, cookieName) {
+OpenAMClient.prototype.getPolicyDecision = function (params, realm, sessionId, cookieName) {
```

Comment:
please reorder the arguments to keep this public method backwards compatible; `realm` should be the last argument and it should have a default value of `/`